package.json

```diff
@@ -104,10 +104,10 @@
 	"dependencies": {
 		"@changesets/cli": "^2.26.2",
 		"@guardian/ophan-tracker-js": "2.1.1",
-		"@guardian/prebid.js": "8.34.0",
 		"@octokit/core": "^6.1.2",
 		"fastdom": "^1.0.11",
 		"lodash-es": "^4.17.21",
+		"prebid.js": "github:guardian/prebid.js#862f625999182d549dddb6598265d7916ecd6e88",
```

Comment:
Is this intentionally a commit hash or should it be a version bump?

Reply:
Yes we have to update to the npm version and undo the dependency updates here once we have merged:

https://github.com/guardian/Prebid.js/pull/147

Reply:
This was just for testing purposes and it should be changed to Prebid.js v8.52.0 once this https://github.com/guardian/Prebid.js/pull/147/ is approved and merged. 

I just noticed now that this is the PR I added to the thread but what is ready for review is https://github.com/guardian/Prebid.js/pull/147/. Sorry for the confusion @emma-imber!

Reply:
Makes sense! Thanks for clarifying both 🙂